rust/src/lib/nispor/veth.rs

```diff
@@ -35,3 +38,33 @@ pub(crate) fn nms_veth_conf_to_np(
         veth_conf
     })
 }
+
+pub(crate) fn process_new_veth_peer(
+    merged_iface: &MergedInterface,
+    merged_ifaces: &MergedInterfaces,
+) -> Option<nispor::IfaceConf> {
+    if merged_iface.current.is_none() && merged_iface.merged.is_up() {
+        if let Some(Interface::Ethernet(apply_iface)) =
+            merged_iface.for_apply.as_ref()
+        {
+            if let Some(peer) =
+                apply_iface.veth.as_ref().map(|v| v.peer.as_str())
+            {
+                // Mark new veth peer as up if not desired
```

Comment:
if not ~desired~ mentioned in desired state. That would be clearer.

Apart from that, the code LGTM, but I'd add a test case if possible.